src/lib/ursadb.py

```diff
@@ -76,7 +76,7 @@ def query(
         file_count = res["result"]["file_count"]
 
         return {
-            "time": (end - start) * 1000,
+            "time": (end - start),
```

Comment:
The resolution is the same, so this should probably still be in ms:

```suggestion
            "time": (end - start) * 1000,
```